docs/content/posts/2023-08-03-connector-updates.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "DynamoDB, MongoDB and OpenSearch externalized connectors"
+date: "2023-08-03T12:00:00Z"
+authors:
+- elphastori: 
+  name: "Elphas Toringepi"
+  twitter: "elphastori"
+aliases:
+- /news/2023/08/03/externalized-connectors.html
+---
+
+## New connectors
```

Comment:
Let's add a section to thank the contributors, like we do for Flink releases. We can run the git commands to pull the list of contributors for each connector. 
```
git log --pretty="%an%n%cn" <fromCommit>..<toCommit> \| sort \| uniq \| tr "\n" "," \| sed 's/,/, /g'
```

Reply:
Thanks, I've listed the contributors.